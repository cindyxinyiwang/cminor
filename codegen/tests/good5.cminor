// functions

func1: function void () = {
	return;

}

func2: function integer (i: integer, j: boolean) = {
	// prototype allowed
	func3: function void (i: integer, j:integer);
	i = 1;
	j = true;
	return i;
	func3(1, 1);
	func1();
}