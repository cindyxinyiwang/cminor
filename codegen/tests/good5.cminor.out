i resovles to param 0
j resovles to param 1
i resovles to param 0
func3 resovles to local 0
func1 resovles to global func1
